ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/ObjectFieldTemplate.js

```diff
@@ -13,11 +13,11 @@ import { ERROR_COLOR } from '../../../../constants/colors';
 import { iconMedium, iconSmall } from '../../../../css/icons.styles';
 import { calculateGrid } from '../helper';
 
-const styles = (theme) => ({
+const styles = () => ({
   objectFieldGrid: {
     padding: '.5rem',
     paddingTop: '0.7rem',
-    backgroundColor: theme.palette.type === 'dark' ? '#363636' : '#ffffff',
+    // backgroundColor: theme.palette.type === 'dark' ? '#363636' : '#ffffff',
```

Comment:
You have tested this Meshmap right :)

Reply:
yes , you can test  a open pr in extensions which has this following change